src/GitExtensions.PluginTemplate/GitExtensions.PluginTemplate.csproj

```diff
@@ -49,10 +49,10 @@
   <PropertyGroup>
     <NuspecFile>$(MSBuildThisFileDirectory)$(MSBuildProjectName).nuspec</NuspecFile>
     <Authors>Maraf</Authors>

```

Comment:
After the "theoretical" discussion about author names in https://github.com/gitextensions/gitextensions.extensibility/issues/3, we have to set for an author name for the first time.
I think it should be either @maraf 's real name or "Git Extensions". Since this is a template that others should follow, I favor "Marek Fišera" a bit over "Git Extensions".
@maraf @RussKie what do you think?

Reply:
I find this a bit hard. It's definitely a value that should be changed.
- If someone forgets, "Git Extensions" makes more sense for me.
- But if someone sees "Git Extensions", he might think that "GE wants to steel your contribution" .

I'm thinking about some virtual person, like "Jon Doe".

Reply:
How about "Anonymous author of a Git Extensions plugin"? 😄

Reply:
```
<Authors>YOUR NAME HERE</Authors>
<Company>YOUR ORGANIZATION HERE</Company>
```
This should make it pretty clear that you are supposed to change these tags.

Reply:
done